### PR TITLE
doc/website/blog: blog post describing the upsert API in v0.9.0

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

File: doc/website/blog/2021-08-05-announcing-upsert-api.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+type User struct {
+	ent.Schema
+}
+
+// Fields of the User.
+func (User) Fields() []ent.Field {
+	return []ent.Field{
+		field.String("email_address").Unique(),
+		field.String("full_name"),
+}
```
Comment (giautm) [inline doc/website/blog/2021-08-05-announcing-upsert-api.md:83]: missing a `}`

File: doc/website/blog/2021-08-05-announcing-upsert-api.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+//		SetEmailAddress(v).
+//		OnConflict(
+//			// Update the row with the new values
+//			// the was proposed for insertion.
+//			sql.ResolveWithNewValues(),
+//		).
+//		// Override some of the fields with custom
+//		// update values.
+//		Update(func(u *ent.UserUpsert) {
+//			SetEmailAddress(v+v).
```
Comment (giautm) [inline doc/website/blog/2021-08-05-announcing-upsert-api.md:122]: has extra dot `.` at the end of line

Comment (a8m) [reply doc/website/blog/2021-08-05-announcing-upsert-api.md:122]: :+1

File: doc/website/blog/2021-08-05-announcing-upsert-api.md (hunk 1 above)
Comment (a8m) [reply doc/website/blog/2021-08-05-announcing-upsert-api.md:83]: 👍

**Pull request merged**